### PR TITLE
S2t structure

### PR DESCRIPTION
**Pull request by SteeleSR**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,12 +2,55 @@
...
+                    <p>Correctly guess as many anagrams of countries of the world as possible. </p>
+                    <ul>
+                        <li>On the following page, an anagram will appear and you must enter your answer in the text field below. </li>
+                        <li>If you need a hint, then hover over the HINT box and the two letter country code will appear. </li>
+                        <li>Once you feel ready, press START and you will be taken to the following screen where the game will start immediately. </li>
+                    </ul>
+                    </div>
+                    <div>
+                        <img src="assets/globe.png" alt="globe" />
+                    </div>
```
Comment (xz-dice) [inline index.html]: Tested without the div and nothing changed - may want to remove this?

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+.instructionMain {
+    display: flex;
+    align-items: center;
+}
+
+.buttons {
+    display: flex;
+    justify-content: space-evenly;
+    width: 100%;
```
Comment (xz-dice) [inline styles.css]: Doesn't seem to do anything.

File: index.html [hunk 3]
```diff
@@ -2,12 +2,55 @@
...
+                </div>
+                <div class="buttons">
+                     <button id="startButton">Start Game</button>
+                </div>
+            </section>
+
+            <section id="mainScreen">
+                <div class="gameNav">
+                </div>
```
Comment (xz-dice) [inline index.html]: Empty div (will this be used in future stories?)

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+
+.gameNav {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.mainGame {
+    display: flex;
```
Comment (xz-dice) [inline styles.css:46]: When these properties are all deselected, there's no change in the browser - is this future coding or can it be removed?

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    flex-direction: column;
+    align-items: center;
+}
+
+.anagramInput {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```
Comment (xz-dice) [inline styles.css]: Doesn't seem to do anything?

**Review by xz-dice — CHANGES_REQUESTED** [review]
Looking good so far! Just a few tweaks/question.

File: styles.css (hunk 4 above)
Comment (SteeleSR) [reply styles.css:46]: I've removed justify content, however, when I remove display: flex and flex-direction: column the layout goes to the left and messes up.

**Pull request merged**